sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -26,12 +27,14 @@
 import com.azure.cosmos.util.CosmosPagedFlux;
 import com.microsoft.data.encryption.cryptography.EncryptionKeyStoreProvider;
 import io.netty.handler.codec.http.HttpResponseStatus;
+import org.testng.Assert;
```

Comment:
we use assrtj for test assertion in the java sdk. to be consistent with the other tests please use assertj not testng for the assertion.

Reply:
Removed, and added the correct assertion usage to be consistent with the rest of the test cases.